### PR TITLE
Issues 10-13 implement enum classes

### PR DESCRIPTION
**Pull request by krzhld**
Added enum classes:

- Delivery
- Food
- Song
- WebTask

### CODE REVIEW

**Review by Bazalii — DISMISSED** [review]
Everything is fine

**Review by Bazalii — APPROVED** [review]
Thanks!

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Song.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package android.clicker.school_live_simulator
+
+enum class Song(val happiness: Int) {
```
Comment (Ferrociraptus) [inline Project_source/app/src/main/java/android/clicker/school_live_simulator/Song.kt:3]: Enum field should be named as salary, not happiness.

**Review by Ferrociraptus — CHANGES_REQUESTED** [review]
You should change one enum field name. 
All in all, it is fine.

**Review by Bazalii — APPROVED** [review]
You did what @Ferrociraptus wanted. Everything is fine

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Song.kt (hunk 1 above)
Comment (krzhld) [reply Project_source/app/src/main/java/android/clicker/school_live_simulator/Song.kt]: Fixed

**Review by Ferrociraptus — APPROVED** [review]
All is fine.

**Comment by Bazalii** [comment]
> All is fine.

POGGERS , first merged PR

**Pull request merged**